tenacity/before_sleep.py

```diff
@@ -50,12 +50,8 @@ def log_it(retry_state: "RetryCallState") -> None:
 
         logger.log(
             log_level,
-            "Retrying {0} in {1} seconds as it {2} {3}.".format(
-                _utils.get_callback_name(retry_state.fn),
-                getattr(retry_state.next_action, "sleep"),
```

Comment:
`getattr` here not protect from `AttributeError` exceptions since no default value specified -> use attribute directly